src/main/groovy/jenkins/automation/rest/RestApiScriptRunner.groovy

```diff
@@ -3,7 +3,6 @@
 package jenkins.automation.rest
 
 import javaposse.jobdsl.dsl.DslScriptLoader
-import javaposse.jobdsl.dsl.ScriptRequest
```

Comment:
not sure about this?

Reply:
Unused import that got cleaned up by idea.

Reply:
👍